src/analytics/analyticsEvents.ts

```diff
@@ -299,8 +299,13 @@ export class AnalyticsEvents {
     }
     let isSynapseUser = false;
     if (score >= requiredScore && requiredScore != 0) {
+      requiredScore = null;
       isSynapseUser = true;
     }
+    if (requiredScore > score && requiredScore != 0) {
+      isSynapseUser = true;
+      nextAssessment = 'null';
```

Comment:
The user fails here, but we still need to send the requiredScore and nextAssessment as null @amitsinghsutara is that the case in this condition for sure? 🎉